src/camps/migrations/0034_auto_20200219_2008.py

```diff
@@ -0,0 +1,53 @@
+# Generated by Django 3.0.3 on 2020-02-19 19:08
```

Comment:
I have a pet peeve that I am almost writing a pre-commit hook for: Migrations should always have verbose names. But maybe it's a bit late in this case :D

Reply:
Another thing, although I'm not sure, but I think there should be ways to define non-db-related field options so they don't prompt migrations when changed. Perhaps dynamically.

Reply:
It is not too late to rename the migrations. Django supports changing the name as long as the 00xx_ prefix number stays the same. Will fix.

Reply:
If you know of a way to do this dynamically I would be interested in learning it. Currently we are forced to tie static code (the permissions sets in camps.models.Permission) with dynamic DB data (the teams) which is clearly not ideal. It works for now, but something more elegant would be nice.